client/src/app/components/user-settings/user-settings.component.ts

```diff
@@ -170,12 +170,12 @@ export class UserSettingsComponent {
   save(value: UserSettingKeyValue) {
     this.http
       .post(
-        `${environment.apiUrl}/accounts/${this.authService.currentUser?.id}/settings`,
+        `${environment.apiUrl}/accounts/${this.userService.currentUser?.id}/settings`,
         value
       )
       .subscribe({
         next: (_) => {
-          this.authService.setUserSettings(value);
+          this.userService.setSettings(value);
```

Comment:
Please rename `value` to settings. My first-glance understanding was that you set an individual setting value here.